docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1026,201 +1026,134 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Action:**
 
-  No action required. 
-
-
-
-### ZWED0289I
-
-  JarMgr with id=%s invoked to startup with config=%s
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
+  No action required.
 
 
 
 ### ZWED0290I
 
-  Plugin (%s) loaded. Version: %s. Successful: %s% (%s/%s) Attempted: %s% (%s/%s)
-
-  **Reason:**
-
-  TODO
-
-  **Action:**
-
-  TODO
-
-
-
-### ZWED0291I
-
-  Server is ready at %s Plugins successfully loaded: %s% (%s/%s)
+  Plugin *event.data.identifier* loaded. Version: *event.data.pluginVersion*. Successful: _(pluginsLoaded/event.count)*100)_ (*pluginsLoaded*/*event.count*) Attempted: _(pluginCount/event.count)*100)_ (*pluginsCount*/*event.count*)
 
   **Reason:**
 
-  TODO
+  Plugin event data identifier loaded successfully. Version is event data pluginVersion.
```

Comment:
The reason is a little awkward here. What this really means is "Plugin with _plugin identifier_ loaded, with version _pluginVersion_. The server attempted to load a total of _pluginsAttempted_ with _pluginsLoaded_ plugins successfully loaded.